android/src/main/java/com/inkapplications/android/Views.kt

```diff
@@ -0,0 +1,65 @@
+package com.inkapplications.android
+
+import android.os.Handler
+import android.os.Looper
+import android.view.animation.AlphaAnimation
+import android.view.View
+
+/**
+ * Set the visibility of a view.
+ *
+ * This works like Android's `setVisibility` but takes a boolean instead
+ * of a `View` integer constant.
+ * Use this to replace unwieldy if statements when binding views.
+ *
+ * Example:
+ *
+ *     fun toggleViews(show: Boolean) {
+ *         my_view.setVisibility(show)
+ *         my_invisible_view.setVisibility(show, View.INVISIBLE)
+ *     }
+ *
+ * @param visible Whether the view should be displayed or not.
+ * @param hiddenState Optional parameter to control how the view is hidden when [visible] is false.
+ */
+fun View.setVisibility(visible: Boolean, hiddenState: Int = View.GONE) {
```

Comment:
Isn't this already in androidx?

Reply:
[Citation Needed]?

Reply:
Ever since they moved it out of github I have no idea how to find anything in it :(